web/app/view/MapController.js

```diff
@@ -288,9 +290,42 @@ Ext.define('Traccar.view.MapController', {
         if (record) {
             if (record instanceof Traccar.model.Device) {
                 this.fireEvent('selectDevice', record, false);
+                this.showAttributesView(feature);
             } else {
                 this.fireEvent('selectReport', record, false);
             }
         }
+    },
+
+    clickMap: function () {
+        this.hidePopup();
+    },
+
+    showAttributesView: function (feature) {
+        var coordinates = feature.getGeometry().getCoordinates();
+        var popupOverlay = this.getView().getPopupOverlay();
+        var record = feature.get('record');
+
+        popupOverlay.setPosition(coordinates);
+
+        Ext.create('Traccar.view.PopupWindow', {
+            title: Strings.sharedAttributes,
+            modal: false,
+            width: 250,
+            height: 250,
+            resizable: false,
+            draggable: false,
+            renderTo: Ext.get('popup'),
+            items: {
+                xtype: 'attributesView',
```

Comment:
Most of the time attributes are empty. Are you sure that's what you want to show?